config/subsidy-application-management/forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl

```diff
@@ -0,0 +1,205 @@
+      sh:path ( schema:contactPoint schema:jobTitle ) ;
+      sh:resultMessage "Dit veld is verplicht."@nl
+    ] ;
+    form:displayType displayTypes:defaultInput ;
+    sh:group ext:contactPg .
+
+
+##########################################################
+#  property-group: Application
+##########################################################
+ext:applicationPg a form:PropertyGroup;
+    sh:name "Aanvraag" ;
+    sh:order 2 .
+
+  ##########################################################
+  #  field: application form
+  ##########################################################
+  ext:applicationFormF a form:Field ;
+    sh:order 1 ;
+    sh:name "Laad hier het ingevuld aanvraagformulier op." ;
+    form:help """Het aanvraagformulier vindt u terug op deze <a href="https://www.vlaanderen.be/stedenbeleid/stadsvernieuwing-in-vlaanderen" target="_blank">webpagina</a>.""" ;
+    sh:path ( lblodSubsidie:urbanRenewalApplicationForm dct:hasPart ) ;
+    form:validations
+    [ a form:RequiredConstraint ;
+      form:grouping form:Bag ;
+      sh:resultMessage "Dit veld is verplicht."@nl;
+      sh:path ( lblodSubsidie:urbanRenewalApplicationForm dct:hasPart )
+    ] ;
+    form:displayType displayTypes:files ;
+    sh:group ext:applicationPg .
+    
+
+  ##########################################################
+  #  hidden field: application form
+    # Hidden field required for all variations of URL or FILE
+    # input field which require validation.
+    # It makes sure there is a type attached to hasPart object.
+    # This enables correct validation in both front and backend.
+  ##########################################################
+  ext:hiddenApplicationFormF a form:Field ;
+    sh:order 2 ;
+    sh:name "Type RemoteDataObject or FileDataObject [hidden input]" ;
+    sh:path ( lblodSubsidie:urbanRenewalApplicationForm dct:hasPart rdf:type );
+    sh:group ext:applicationPg .
+
+  ##########################################################
+  #  field: attachments
+  ##########################################################
+  ext:attachmentsF a form:Field ;
+    sh:order 3 ;
+    sh:name "Laad hier alle relevante bijlagen op." ;
+    sh:path ( lblodSubsidie:urbanRenewalAttachments dct:hasPart ) ;
+    form:displayType displayTypes:files ;
+    sh:group ext:applicationPg .
+    
+
+  ##########################################################
+  #  hidden field: attachments
+    # Hidden field required for all variations of URL or FILE
+    # input field which require validation.
+    # It makes sure there is a type attached to hasPart object.
+    # This enables correct validation in both front and backend.
+  ##########################################################
+  ext:hiddenAttachmentsF a form:Field ;
+    sh:order 4 ;
+    sh:name "Type RemoteDataObject or FileDataObject [hidden input]" ;
+    sh:path ( lblodSubsidie:urbanRenewalAttachments dct:hasPart rdf:type );
+    sh:group ext:applicationPg .    
```

Comment:
Is this still needed if the field has no validation?

Reply:
If I remember correctly I am pretty sure yes, has something to do with how we model it in the backend. we have 2 links for a file, a virtual one and a physical one